pkgs/development/python-modules/scramp/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytest }:
+
+buildPythonPackage rec {
+  pname = "scramp";
+  version = "1.1.0";
```

Comment:
```suggestion
  version = "1.1.0";
  disabled = isPy27;
```

to avoid:
```
  Processing ./scramp-1.1.0-py2-none-any.whl
  ERROR: Package 'scramp' requires a different Python: 2.7.17 not in '>=3.5'
```

Reply:
I added: `disabled = !(pythonAtLeast "3.5");`